src/jshint.js

```diff
@@ -2015,7 +2026,7 @@ var JSHINT = (function() {
               funct[v] = s["(global)"] ? "global" : "outer";
               break;
             case "const":
-              setprop(s, v, { unused: false });
+              s[v]["(type)"] = false;
```

Comment:
const is required to have a type set to const in order to check whether the label has been modified.. var/let use a type of "unused" which becomes "var" once used. function begins as "unction" and becomes "function" when used.

I could have started "const" as "onst" and then once used promoted to "const" but then I would also have to look for type "const" || "onst" in a few places which I didn't like.

Personally I'd prefer to migrate var,let and function to have types "var","let","function" and use a boolean - what do you think? I was going to do that as part of the PR I am working on after this.. but whatever you feel is more maintainable?

a) leave as is
b) use "onst" for unused "const"
c) move unused and unction to this way in this PR
d) ""                         ""                              in a later PR
e) something better you can think of?